src/utils/material.js

```diff
@@ -4,6 +4,8 @@
  * @param {object} shader - A-Frame shader instance.
  * @param {object} data
  */
+var THREE = require('../lib/three');
```

Comment:
We should put this at the very top since it's separating a function from its docstring